contrib/gomodule/redigo: Don't pass nil contexts around in withSpan

abustany opened the pull request:
This fixes issue #710.

gbbr commented on contrib/gomodule/redigo/redigo.go:
```diff
@@ -136,6 +136,15 @@ func withSpan(do func(commandName string, args ...interface{}) (interface{}, err
 	}
 
+	if ctx == nil {
+		// no context (or a nil context) was passed by the caller, but withSpan
+		// and the tracer functions rely on having a context.
+		//
```
I think we can remove this part of the comment.

gbbr commented on contrib/gomodule/redigo/redigo.go:
```diff
@@ -136,6 +136,15 @@ func withSpan(do func(commandName string, args ...interface{}) (interface{}, err
+		// was closed.
+		ctx = context.Background()
+	}
+
```
Can we please remove this empty line?

gbbr reviewed (commented):
Thanks for doing this

abustany commented:
Thanks for the quick review! I've [addressed the comments](https://github.com/DataDog/dd-trace-go/compare/8376dd1e66e4fadc283f3064371222cb731ef734..eb4207e2ae9edea746be0d78da7248f3de502946).

gbbr reviewed (approved):
Thanks a lot!

jontonsoup commented:
is there a schedule on when this will be cut into a release? it prevents us from upgrading to 15 currently :)
